File: ParkingPlaceServer/ParkingPlaceServer/DTO/ParkingPlacesUpdatingDTO.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ParkingPlaceServer.DTO
{
	public class ParkingPlacesUpdatingDTO
	{
		[JsonProperty("changes")]
		public List<ParkingPlaceChangesDTO> Changes { get; set; }

		[JsonProperty("initials")]
		public List<ParkingPlacesInitialDTO> Initials { get; set; }

		public ParkingPlacesUpdatingDTO(List<ParkingPlaceChangesDTO> changes, List<ParkingPlacesInitialDTO> initials)
		{
			Changes = changes;
			Initials = initials;
		}
	}
}

File: ParkingPlace/app/src/main/java/com/rmj/parking_place/dto/ParkingPlacesUpdatingDTO.java
package com.rmj.parking_place.dto;

import java.util.List;

public class ParkingPlacesUpdatingDTO {
    private List<ParkingPlaceChangesDTO> changes;
    private List<ParkingPlacesInitialDTO> initials;

    public ParkingPlacesUpdatingDTO() {

    }

    public List<ParkingPlaceChangesDTO> getChanges() {
        return changes;
    }

    public void setChanges(List<ParkingPlaceChangesDTO> changes) {
        this.changes = changes;
    }

    public List<ParkingPlacesInitialDTO> getInitials() {
        return initials;
    }

    public void setInitials(List<ParkingPlacesInitialDTO> initials) {
        this.initials = initials;
    }
}
